lib/src/main/java/com/auth0/spring/security/api/JwtWebSecurityConfigurer.java

```diff
@@ -87,6 +124,43 @@ public static JwtWebSecurityConfigurer forHS256(String audience, String issuer,
         return new JwtWebSecurityConfigurer(audience, issuer, provider);
     }
 
+    /**
```

Comment:
Have the other signatures receiving a single String for "issuer", delegate the call to these.

Reply:
Agreed!